xen/arch/x86/tpm.c

```diff
@@ -985,4 +978,136 @@ void __stdcall tpm_extend_mbi(uint32_t *mbi)
     tpm_hash_extend(DRTM_LOC, DRTM_DATA_PCR, (uint8_t *)mbi, *mbi,
                     TXT_EVTYPE_MBI, NULL, 0);
 }
+#else
+static struct slr_entry_policy *tpm_get_policy(void)
```

Comment:
This function has nothing to do with TPM.
```suggestion
static struct slr_entry_policy *slr_get_policy(void)
```

Reply:
https://github.com/TrenchBoot/xen/compare/bab60f90c52351add03aec69881f03a9c30cda6a..d820e3dac46047e7e81db6fdc30d72041e9d208e#diff-5861d423276bd98c31fb2a501569110297cbe6488842e0359435a3af795c24aeL982-R982